acceptance/lib/acceptance/install_ruby.rb

```diff
@@ -0,0 +1,7 @@
+def install_ruby_from_source(host, version="2.4.4")
+  url = "https://artifactory.delivery.puppetlabs.net/artifactory/generic__buildsources/buildsources/ruby-#{version}.tar.gz"
```

Comment:
I'd like to see this in an environment variable. It doesn't look like we have anything in Jenkins yet. Maybe just a way to override with an environment variable if folks want to run this outside our VPN.